### DISCUSSION TITLE
ビルドができません！特にスクリプトも書いていないので原因がわからないです

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#133] [Question-original]
## (1) 実現したいこと
オートレイアウト
## (2) 実現するために自分が試した内容とその結果
教科書12ページまでやりました
## (3) 調べた内容や資料
教科書のAutoLayout
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
Buildできないです！（ずっとロードされている？）
![image](https://user-images.githubusercontent.com/75009903/217884119-e283aeb3-fded-4232-b021-bf6152a00ca2.png)
## (5) GitHubのリンク
https://github.com/KOTARO1209/Autolayout

**Comment by appleEmily** [comment]
この教科書は、buildせずに確認してほしいです！
教科書的には、添付した画像のように、デバイスサイズを変更して確認します。
(今回、じねんじょはXcodeをアップデートしていなかったようなので、previewが表示できていますが、最新版からpreviewがなくなっています。教科書も昨日くらいに新しいものになったので、これに取り組んでくれた時は、古い教科書になってたかも><変更点はそこだけなので、許して〜)

buildできない理由としては、Storyboardのファイルを今回たくさん用意していて、build対象Storyboardをちゃんと指定する必要があるからって感じです。
<img width="990" alt="image" src="https://user-images.githubusercontent.com/49332380/217977542-e42d6be3-387b-4f8e-bad6-81d1bbce0956.png">
